plugin/src/main/java/io/jenkins/plugins/casc/impl/configurators/HeteroDescribableConfigurator.java

```diff
@@ -117,6 +117,18 @@ public CNode describe(T instance, ConfigurationContext context) {
                 }).getOrNull();
     }
 
+    @CheckForNull
+    public void describeStructure(T instance, ConfigurationContext context) {
+        lookupConfigurator(context, instance.getClass())
```

Comment:
@casz I am trying to understand the inital implementation of the describe function.
Would this return to me all of the configurators given a context.

Reply:
it provides it looks up the configurator and then you could loop over that and ask for it's confiurators